arbeitszeit_flask/templates/macros/plan_details.html

```diff
@@ -112,14 +122,23 @@ <h1 class="title">
                     <div class="tile">
                         <div class="tile is-parent is-vertical">
                             <article class="tile is-child notification has-background-danger-light">
+                                <div class="has-text-right has-text-info-dark">
+                                    {{ info_tooltip(id="id_fixed_means", text=gettext("This is sometimes also called depreciation, e.g. if you plan to use a machine that cost 10000 hours to produce, for 10 years, and you have ten one-year-plans, then for this fixed mean of production, the value would be 1000.")) }}
+                                </div>
                                 <p class="title is-4">{{ view_model.details.means_cost[0] }}</p>
                                 <p class="subtitle">{{ view_model.details.means_cost[1] }}</p>
                             </article>
                             <article class="tile is-child notification has-background-danger-light">
+                                <div class="has-text-right has-text-info-dark">
+                                    {{ info_tooltip(id="id_liquid_means", text=gettext("Here, recurring costs that can vary from plan to plan are summarized, e.g. electricity, rent for production halls, maintenance of machines etc.")) }}
```

Comment:
I think I would propose to delete "that can vay from plan to plan", because also fixed means of production can vary from plan to plan (not every machine will be used for every product). 
 Perhaps we should mention that we refer to marxian "circulating means of production" here? Perhaps we should even rename "liquid" to "circulating". See the wiki page: https://en.wikipedia.org/wiki/Circulating_capital. Not quite sure why we chose "liquid" from the beginning.  

 See Grundprinzipien 1930:
 "As is well known the Marxist category "means of production" comprises machinery and buildings (fixed means of production), and also raw materials and auxiliary materials (circulating means of production). If now we use for fixed means of production the letter p and for the circulating means of production the letter c, then the formula takes on the following form: ( p + c ) + L = product" https://www.marxists.org/subject/left-wing/gik/1930/03.htm
 In German: "Wie bekannt, umfaßt die Marx'sche Kategorie "Produktionsmittel" , Maschinen, Gebäude (feste Produktionsmittel), als auch die Rohmaterialien und Hilfsstoffe (zirkulierende Produktionsmittel). "

 In Grundprinzipien 1935:
 "Each company calculates how much working time is spent on its product, i.e., it determines how many working hours of "fixed" means of production (machines and buildings), how many working hours of "circulating" means of production (raw materials and consumables) and how many directly consumed working hours flow into the product." (p. 111f., https://arbeitszeit.noblogs.org/files/2023/04/GIC-Fundamental-Principles-2.-Ed.1935-1.pdf)